shell/client/shell.html

```diff
@@ -75,12 +75,13 @@
   {{/if}}
   {{#if currentUser}}
     {{>sandstormTopbarItem name="notifications" topbar=globalTopbar
-                           template="notifications" popupTemplate="notificationsPopup"}}
+                           template="notifications" popupTemplate="notificationsPopup"
+                           priority=-1}}
   {{/if}}
   {{!-- admin alerts --}}
   {{#with adminAlert}}
     {{#if adminAlertIsTooLarge}}
-      {{#sandstormTopbarItem name="admin-alert" topbar=globalTopbar}}
+      {{#sandstormTopbarItem name="admin-alert" topbar=globalTopbar priority=-2}}
```

Comment:
It does! Priority determines the ordering of the topbar items in the HTML, which implies determining the ordering in the top bar. I was seeing a bug where the ordering was non-deterministic. I think this bug predated my changes. Anyway, assigning priorities fixes it.